src/containers/org/OrgDataSetContainer.js

```diff
@@ -0,0 +1,165 @@
+/*
+ * @flow
+ */
+
+import React, { useEffect, useMemo } from 'react';
+
+import { Map, getIn } from 'immutable';
+import { AppContentWrapper, AppNavigationWrapper, Typography } from 'lattice-ui-kit';
+import { useRequestState } from 'lattice-utils';
+import { useDispatch, useSelector } from 'react-redux';
+import { Route, Switch } from 'react-router';
+import { NavLink } from 'react-router-dom';
+import { RequestStates } from 'redux-reqseq';
+import type {
+  EntitySet,
+  Organization,
+  UUID,
+} from 'lattice';
+import type { RequestState } from 'redux-reqseq';
+
+import DataSetDataContainer from './DataSetDataContainer';
+import DataSetMetaContainer from './DataSetMetaContainer';
+
+import {
+  CrumbItem,
+  CrumbLink,
+  Crumbs,
+  NavContentWrapper,
+  Spinner,
+  StackGrid,
+} from '../../components';
+import { GET_OR_SELECT_DATA_SET, getOrSelectDataSet } from '../../core/edm/actions';
+import { EDM } from '../../core/redux/constants';
+import {
+  selectAtlasDataSets,
+  selectEntitySets,
+  selectOrganization,
+} from '../../core/redux/selectors';
+import { Routes } from '../../core/router';
+
+const OrgDataSetContainer = ({
+  dataSetId,
+  organizationId,
+} :{|
+  dataSetId :UUID;
+  organizationId :UUID;
+|}) => {
+
+  const dispatch = useDispatch();
+
+  // BUG: this is a bug because getOrSelectDataSets is called in the background
+  const getOrSelectDataSetRS :?RequestState = useRequestState([EDM, GET_OR_SELECT_DATA_SET]);
```

Comment:
seems a little unclear what part of this is the bug?

Reply:
In an earlier version, I was calling `getOrSelectDataSets` (plural), which was the problem. I've since changed it to `getOrSelectDataSet` (singular) and I guess I forgot to remove this comment.